src/commons/actor.rs

```diff
@@ -117,7 +117,11 @@ pub struct Actor {
     is_user: bool,
     attributes: Attributes,
     new_auth: Option<Auth>,
+
+    #[cfg_attr(not(feature = "multi-user"), allow(dead_code))]
```

Comment:
is `allow(dead_code)` still needed when the `feature` notations are there as well?

Reply:
Yes because you can't use `#[cfg(feature(multi-user))]` here as that would remove the fields entirely and then break lots of other code, and we don't want to mark all those other places as multi-user only as that makes the code less readable, at least that's what we discussed in the past.